Readme fix and teacher profile schema (#7)
iankameron/eibar-backend: readme.md

## What is Eibar and what is the role of this repo?

Eibar is the "Uber" for freelance, in-person language tutors. It helps teachers and students who sign up as users of the Eibar app to find each other for in-person language lessons.

> Anyone learning a language knows how valuable a face to face tutor can be. They also know how hard it can be to find one. Especially at a time and in a location that is suitable. Eibar seeks to keep students motivation high by making it easy to connect with tutors, and expanding the language learning market 

Reviewer: face to face should be hyphenated
NAB

Reviewer: please revise overall

This repo is the heart of the Eibar system, as it is the API backend that holds (will hold?) all user, lesson, and payment data and serve that data to the various other Eibar consumers of that data: web application, iOS, and Android apps.

## Getting started

> The Eibar development environment, which is possible to recreate by installing the following prerequisites

Reviewer: ```suggestion
The Eibar development environment can be recreated by installing the following prerequisites.
```
